app/gui2/src/components/GraphEditor/widgets/WidgetPort.vue

```diff
@@ -117,6 +118,15 @@ function updateRect() {
   if (portRect.value != null && localRect.equals(portRect.value)) return
   portRect.value = localRect
 }
+
+const keyboard = injectKeyboard()
+
+const enabled = computed(() => {
+  const input = props.input.value
+  const isConditional = input instanceof Ast.Ast && tree.conditionalPorts.has(input.id)
+  return !isConditional || keyboard.mod
+})
+watch(enabled, () => tree.emitTargetablePortsChanged())
```

Comment:
* We could also set "targetablePortChanged" when the port position changes (updatePortRect I believe).
* The selection is provided for entire Graph Editor - I think we could just inject it here and call "targetablePortChanged" there instead of propagating the signal through entire tree?